hibernate-core/src/main/java/org/hibernate/tool/schema/spi/Exporter.java

```diff
@@ -22,13 +24,45 @@
 public interface Exporter<T extends Exportable> {
 	String[] NO_COMMANDS = new String[0];
 
+	/**
+	 * Get the commands needed for creation.
+	 *
+	 * @return The commands needed for creation scripting.
+	 * @deprecated Will be removed in favor of the variant accepting {@link SqlStringGenerationContext}
+	 * @see #getSqlCreateStrings(Exportable, Metadata, SqlStringGenerationContext)
+	 */
+	@Deprecated
+	default String[] getSqlCreateStrings(T exportable, Metadata metadata) {
+		final SqlStringGenerationContext context = SqlStringGenerationContextImpl.fromConfigurationMap(
+				metadata.getDatabase().getJdbcEnvironment(),
+				metadata.getDatabase(),
+				metadata.getDatabase().getServiceRegistry().getService( ConfigurationService.class ).getSettings()
+		);
+		return getSqlCreateStrings( exportable, metadata, context );
+	}
 	/**
 	 * Get the commands needed for creation.
 	 *
 	 * @return The commands needed for creation scripting.
 	 */
 	String[] getSqlCreateStrings(T exportable, Metadata metadata, SqlStringGenerationContext context);
 
+	/**
+	 * Get the commands needed for dropping.
+	 *
+	 * @return The commands needed for drop scripting.
+	 * @deprecated Will be removed in favor of the variant accepting {@link SqlStringGenerationContext}
+	 * @see #getSqlDropStrings(Exportable, Metadata, SqlStringGenerationContext)
+	 */
+	@Deprecated
+	default String[] getSqlDropStrings(T exportable, Metadata metadata) {
```

Comment:
Just a warning: you're restoring backwards compatibility for callers of this method, but not for implementors. This method will never get called by Hibernate ORM.